### PR TITLE
Bjorncs/apache commons libraries cleanup

### PR DESCRIPTION
**Pull request by bjorncs**
Part of #8788

### CODE REVIEW

**Comment by bjorncs** [comment]
I will continue the cleanup in an upcoming PR. There is more unnecessary usage of Apache commons libraries in other modules + I will add some utility classes to vespajlib (e.g stack trace to string).

**Review by baldersheim — APPROVED** [review]
Johooooooo :)

File: jdisc_http_service/src/test/java/com/yahoo/jdisc/http/server/jetty/servlet/ServletTestBase.java [hunk 1]
```diff
@@ -27,15 +24,27 @@
...
  * @author Tony Vaagenes
  * @author bakksjo
  */
 public class ServletTestBase {
-    private ImmutableMap<Pair<ComponentId, String>, HttpServlet> servlets = ImmutableMap.of(
-            ImmutablePair.of(TestServlet.ID, TestServlet.PATH), new TestServlet(),
-            ImmutablePair.of(NoContentTestServlet.ID, NoContentTestServlet.PATH), new NoContentTestServlet());
+
+    private static class ServletInstance {
+        final ComponentId componentId; final String path; final HttpServlet instance;
```
Comment (gjoranv) [inline jdisc_http_service/src/test/java/com/yahoo/jdisc/http/server/jetty/servlet/ServletTestBase.java]: Why not use linebreaks?

**Comment by gjoranv** [comment]
Nice! Extra honours for application and jdisc_http_service.

**Pull request merged**